src/test/ui/numeric/uppercase-base-prefix-invalid-no-fix.stderr

```diff
@@ -0,0 +1,50 @@
+error: invalid suffix `X1a3` for number literal
+  --> $DIR/uppercase-base-prefix-invalid-no-fix.rs:4:13
+   |
+LL |     let a = 123X1a3;
+   |             ^^^^^^^ invalid suffix `X1a3`
+   |
+   = help: the suffix must be one of the numeric types (`u32`, `isize`, `f32`, etc.)
+
+error: invalid suffix `O123` for number literal
+  --> $DIR/uppercase-base-prefix-invalid-no-fix.rs:9:13
+   |
+LL |     let b = 456O123;
+   |             ^^^^^^^ invalid suffix `O123`
+   |
+   = help: the suffix must be one of the numeric types (`u32`, `isize`, `f32`, etc.)
+
+error: invalid suffix `B101` for number literal
+  --> $DIR/uppercase-base-prefix-invalid-no-fix.rs:14:13
+   |
+LL |     let c = 789B101;
+   |             ^^^^^^^ invalid suffix `B101`
+   |
+   = help: the suffix must be one of the numeric types (`u32`, `isize`, `f32`, etc.)
+
+error: invalid suffix `XYZ` for number literal
+  --> $DIR/uppercase-base-prefix-invalid-no-fix.rs:19:13
+   |
+LL |     let d = 0XYZ;
+   |             ^^^^ invalid suffix `XYZ`
+   |
+   = help: the suffix must be one of the numeric types (`u32`, `isize`, `f32`, etc.)
```

Comment:
This is the case where the error message should suggest `0xYZ` as an alternative, like this:
```
error: invalid base prefix for number literal
 --> a.rs:5:13
  |
5 |     let d = 0XYZ;
  |             ^^^^ help: try making the prefix lowercase (notice the capitalization): `0xYZ`
  |
  = note: base prefixes (`0xff`, `0b1010`, `0o755`) are lowercase
```

Reply:
Sorry, I was mistaken about this.